codigo/trabajoFinalJava/src/trabajofinaljava/TrabajoFinalJava.java

```diff
@@ -0,0 +1,275 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package trabajofinaljava;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author USUARIO
+ */
+public class TrabajoFinalJava {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        // TODO code application logic here
+        Scanner entrada = new Scanner(System.in);
+        String salida;
+        String ciclo;
+        String cadenaFinal="";
+        int contador = 0;
+        int opciones;
+        boolean bandera = true;
+        while (bandera) {
+            System.out.printf("Selecciona 1 y creamos una cuenta de Facebook\n"
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.